source/assets/sass/vendor/_syntax-accessible.scss

```diff
@@ -1,273 +1,99 @@
 @use '../functions';
 
-// Stylesheet pulled from Rogue stylesheet with colours updated for a11y
+/**
+ * Based on the GHColors theme by Avi Aryan (http://aviaryan.in)
+ * Inspired by Github syntax coloring
+ */
 
-.highlight table td { padding: functions.sl-px-to-rem(5px); }
-.highlight table pre { margin: 0; }
-.highlight .cm {
-  color: #656556;
-  font-style: italic;
-}
-
-.highlight .cp {
-  color: #999999;
-  font-weight: bold;
-}
-
-.highlight .c1 {
-  color: #656556;
-  font-style: italic;
-}
-
-.highlight .cs {
-  color: #999999;
-  font-weight: bold;
-  font-style: italic;
-}
-
-.highlight .c, .highlight .cd {
-  color: #006666;
-  font-style: italic;
-}
-
-.highlight .err {
-  color: #a61717;
-  background-color: #e3d2d2;
-}
-
-.highlight .gd {
-  color: #000000;
-  background-color: #ffdddd;
-}
-
-.highlight .ge {
-  color: #000000;
-  font-style: italic;
-}
-
-.highlight .gr {
-  color: #aa0000;
-}
-
-.highlight .gh {
-  color: #999999;
-}
-
-.highlight .gi {
-  color: #000000;
-  background-color: #ddffdd;
-}
-
-.highlight .go {
-  color: #888888;
-}
-
-.highlight .gp {
-  color: #555555;
-}
-
-.highlight .gs {
-  font-weight: bold;
-}
-
-.highlight .gu {
-  color: #aaaaaa;
-}
-
-.highlight .gt {
-  color: #aa0000;
-}
-
-.highlight .kc {
-  color: #000000;
-  font-weight: bold;
-}
-
-.highlight .kd {
-  color: #000000;
-  font-weight: bold;
-}
-
-.highlight .kn {
-  color: #000000;
-  font-weight: bold;
-}
-
-.highlight .kp {
-  color: #000000;
-  font-weight: bold;
-}
-
-.highlight .kr {
-  color: #000000;
-  font-weight: bold;
-}
-
-.highlight .kt {
-  color: #445588;
-  font-weight: bold;
-}
-
-.highlight .k, .highlight .kv {
-  color: #000000;
-  font-weight: bold;
-}
-
-.highlight .mf {
-  color: #006666;
-}
-
-.highlight .mh {
-  color: #006666;
-}
-
-.highlight .il {
-  color: #006666;
-}
-
-.highlight .mi {
-  color: #006666;
-}
-
-.highlight .mo {
-  color: #006666;
-}
-
-.highlight .m, .highlight .mb, .highlight .mx {
-  color: #006666;
-}
-
-.highlight .sb {
-  color: #d14;
-}
-
-.highlight .sc {
-  color: #d14;
-}
-
-.highlight .sd {
-  color: #d14;
-}
-
-.highlight .s2 {
-  color: #d14;
-}
-
-.highlight .se {
-  color: #d14;
-}
-
-.highlight .sh {
-  color: #d14;
-}
-
-.highlight .si {
-  color: #d14;
-}
-
-.highlight .sx {
-  color: #d14;
-}
-
-.highlight .sr {
-  color: #006619;
+code[class*="language-"],
+pre[class*="language-"] {
+  background-color: #f8f8f8;
```

Comment:
@stacyk The colors in this file are only used here. Do you think they should still be turned into variables?

Reply:
This color is also used in source/assets/sass/components/_link-header.scss, I think?

Reply:
Ah yes, that's the color I added to replace a dynamic ERB reference to a Rouge theme color.